src/style/global-style.js

```diff
@@ -34,6 +34,19 @@ button {
   50% { opacity: 0.6 }
   to { opacity: 0.85 }
 }
+
+/* Remove outline for non-keyboard :focus */
+*:focus:not(:focus-visible) {
+  outline: none !important;
+  box-shadow: 0 0 0 rgb(255, 255, 255) !important;
+}
+
+*:focus-visible {
+  outline: 2px solid #286244 !important;
+  box-shadow: 0 0 5px 3px inset #8ec8aa !important;
```

Comment:
I think I'm not a fan of the blur :/ I would use a blur of 0. But also, apparently I was wrong and `outline` works fine with border-radius. So maybe there is no use for box-shadow at all anymore.